doc/ref/language.xml

```diff
@@ -405,7 +410,71 @@ gap> 2 * 2 + 9 = Fibonacci(7) and Fibonacci(13) in Primes;
 true
 ]]></Example>
 <P/>
-For the precedence of operators, see&nbsp;<Ref Sect="Comparisons"/>.
+
+The following table lists all operators by precedence, from highest to
+lowest, and also indicates whether the operator is left associative (aka
+left-to-right) or right associative (aka right-to-left) or neither.
+
+<Index>precedence</Index>
+<Index>associativity</Index>
+<Table Align="ll">
+<Row>
+  <Item><E>operator</E></Item>
+  <Item><E>associativity</E></Item>
+</Row>
+<HorLine/>
+<Row>
+  <Item><E>arithmetic <Ref Sect="Arithmetic Operators"/></E></Item>
+  <Item></Item>
+</Row>
+<Row>
+  <Item>&nbsp;&nbsp;&nbsp;&nbsp;<C>^</C></Item>
+  <Item>none</Item>
+</Row>
+<Row>
+  <Item>&nbsp;&nbsp;&nbsp;&nbsp;unary <C>+</C>, unary <C>-</C></Item>
+  <Item>right-to-left</Item>
+</Row>
+<Row>
+  <Item>&nbsp;&nbsp;&nbsp;&nbsp;<C>*</C>, <C>/</C>, <K>mod</K></Item>
+  <Item>left-to-right</Item>
+</Row>
+<Row>
+  <Item>&nbsp;&nbsp;&nbsp;&nbsp;binary <C>+</C>, binary <C>-</C></Item>
+  <Item>left-to-right</Item>
+</Row>
+<HorLine/>
+<Row>
+  <Item><E>comparison <Ref Sect="Comparisons"/></E></Item>
+  <Item></Item>
+</Row>
+<Row>
+  <Item>&nbsp;&nbsp;&nbsp;&nbsp;<C>=</C> tests for equality, <C>&lt;></C> tests for inequality</Item>
+  <Item>none</Item>
+</Row>
+<Row>
+  <Item>&nbsp;&nbsp;&nbsp;&nbsp;<C>&lt;</C> less than, <C>&lt;=</C> less than or equal, <C>></C> greater than, and <C>>=</C> greater than or equal</Item>
```

Comment:
The comparison operator `in` (mentioned a few paragraphs above) doesn't appear in this table

Reply:
Fixed